Tests/UploadTests.swift

```diff
@@ -518,6 +518,32 @@ class UploadMultipartFormDataTestCase: BaseTestCase {
             XCTFail("Content-Type header value should not be nil")
         }
     }
+    
+    func testThatUploadingMultipartFormDataWithNotExistFileShouldFail() {
+        // Given
+        let urlString = "https://httpbin.org/post"
+        let imageURL = URL(string: "file://not_exists_file.jpg")!
+
+        let expectation = self.expectation(description: "multipart form data upload with wrong file should fail")
```

Comment:
```suggestion
        let expectation = self.expectation(description: "multipart form data upload from nonexistent file should fail")
```

Again, this can be made a bit easier to read.